daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/speedy/Compiler.scala

```diff
@@ -256,7 +256,7 @@ private[lf] final class Compiler(
   def unsafeCompileModule(
       pkgId: PackageId,
       module: Module,
-  ): Iterable[(SDefinitionRef, SExpr)] = {
+  ): Iterable[(SDefinitionRef, SDefinition)] = {
     val builder = Iterable.newBuilder[(SDefinitionRef, SExpr)]
```

Comment:
```suggestion
    val builder = Iterable.newBuilder[(SDefinitionRef, SDefinition)]
```

Reply:
+ push the SDefinition inside the method `topLevelFunction`

Reply:
I considered this, but thought it was cleaner to just wrap it in one place. I’ll give it a try tomorrow though.

Reply:
With this change, you wrap it in only one place.

Reply:
Done.